docs: add a guide for linting and bundling as two separate but chained commands (#1385)
Redocly/redocly-cli: docs/guides/index.md

    - header: Enforce response contents
      link: docs/cli/guides/response-contains-property
      text: How to create a custom rule to enforce response contents.
    - header: Lint and bundle in one command
      link: docs/cli/guides/lint-and-bundle

>       text: Combine lint and bundle together and check both commands succeed.

Reviewer: ```suggestion
      text: Combine lint and bundle commands and require each to succeed.
```

Author: I re-worded this one.
